common/test/acceptance/tests/lms/test_programs.py

```diff
@@ -24,6 +25,14 @@ def setUp(self):
 
         self.programs = ProgramFactory.create_batch(3)
         self.pathways = PathwayFactory.create_batch(3)
+        self.course_runs = [
+            CourseRunFactory.create(programs=[ProgramDescriptionFactory.from_program(program)], **course_run)
+            for program in self.programs
+            for course in program['courses']
+            for course_run in course['course_runs']
+        ]
+        import json
```

Comment:
If you're going to do it in multiple steps, you should do it w/ generator comprehensions (just sub in `()` in place of `[]`), so that you aren't building out multiple lists in memory.